rocketpy/AeroSurface.py

```diff
@@ -1892,3 +1883,115 @@ def allInfo(self):
         self.aerodynamicInfo()
 
         return None
+
+
+class RailButtons(AeroSurface):
+    """Class that defines a generic rail button.
```

Comment:
- What do you mean by generic?
- I would complete with the word "pair" here.
- My final suggestion: "Class to define a rail button pair."